src/components/implementation/no_interface/llbooter/boot_deps.h

```diff
@@ -47,7 +51,7 @@ boot_comp_pgtbl_expand(int n_pte, pgtblcap_t pt, vaddr_t vaddr, struct cobj_head
 	int i;
 	int tot = 0;
 	/* Expand Page table, could do this faster */
-	for (i = 0 ; i < (int)h->nsect ; i++) {
+	for (i = 0; i < (int)h->nsect; i++) {
```

Comment:
Why is this unwanted? 90% of the code base consistently doesn't do spaces in for loops like this. Isn't this just a consistency fix?